### PR TITLE
Wrap a constructor list to avoid lines too long to fit the screen

### PR DESCRIPTION
**Pull request by ababkin**
addresses https://github.com/jaspervdj/stylish-haskell/issues/35

### CODE REVIEW

**Comment by ababkin** [comment]
A test is failing I see, tried fixing it to no avail. Can I get some help with this?

**Comment by jaspervdj** [comment]
The test case expects:

```haskell
import Data.Identity
    (Identity (Identity, runIdentity))
```

as output, but with your changes we are getting the output:

```haskell
import Data.Identity
    ( Identity (Identity,
                                runIdentity)
    )
```

File: lib/Language/Haskell/Stylish/Step/Imports.hs [hunk 1]
```diff
@@ -372,7 +379,7 @@ prettyImport columns Options{..} padQualified padName longest imp
     mapSpecs f = case importSpecs of
         Nothing -> []     -- Import everything
         Just [] -> ["()"] -- Instance only imports
-        Just is -> f $ map (prettyImportSpec separateLists) is
+        Just is -> f $ map (prettyImportSpec columns separateLists (afterAliasBaseLength + 2)) is
```
Comment (jaspervdj) [inline lib/Language/Haskell/Stylish/Step/Imports.hs]: You should use `listPadding'` here rather than `(afterAliasBaseLength + 2)`.

**Comment by jaspervdj** [comment]
I think this code will break when the constructors are not the first thing in the import list.  A better way to do this would be to convert the result type of `prettyImportSpec` to `[String]` so we can also feed it into the `wrap` and `wrapRest` utility functions, that way you wouldn't need to do the computation yourself.

**Pull request closed without merging**